beater/server.go

```diff
@@ -46,6 +46,8 @@ func newServer(config Config, publish successCallback) *http.Server {
 		responseValid.Inc()
 	})
 
+	logp.Info("Starting server on address: %s", config.Host)
```

Comment:
how about "Starting APM server on ..." to make it clear which server?

Reply:
`APM Server` or `apm-server`, please :)

Reply:
@beniwohli I'm actually thinking we should leave out apm part. As otherwise it will get confusing with the overall start of the APM Server.

What about `Start listening on address: ...`

Reply:
Sounds fine :)

Reply:
I suggest simply: `Listening on %s`